pkgs/servers/aws-codeartifact-proxy/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "aws-codeartifact-proxy";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "sktan";
+    repo = "aws-codeartifact-proxy";
+    rev = "v${version}";
+    hash = "sha256-mHXqwH6C6jUighkeRUu4H30z0o+Y0Zh7aE6foGZ2gKY=";
+  } + "/src";
```

```suggestion
  };
```
`sourceRoot` would be preferred I think

---

Thanks, I hadn't been aware of this attribute before. Implemented in b4ea51d6d2fb28ea7bb20dbbe5705d853ad0c6a2.